### DISCUSSION TITLE
All of the vendor files seem to be throwing exceptions

### DISCUSSION THREAD

**Discussion by sathrall** [userfrosting/UserFrosting#1184] [Q&A & Troubleshooting]
I noticed that there are missing vendor files because there's an exception thrown for each of them when any template attempts to render.

>An exception has been thrown during the rendering of a template ("No file could be resolved for the stream path 'assets://vendor/fontawesome-iconpicker/dist/css/fontawesome-iconpicker.css'.").

**Comment by lcharette** [comment]
This is a common problem. See doc:  https://learn.userfrosting.com/troubleshooting/common-problems#an-exception-has-been-thrown-during-the-rendering-of-a-template
